features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1355,6 +1355,15 @@
             <Scopes>internal_application_mgt_update</Scopes>
         </Resource>
 
+        <Resource context="(.*)/api/server/v1/validation-rules(.*)" secured="true" http-method="GET">
+            <Permissions>/permission/admin/manage/identity/claimmgt/metadata/view</Permissions>
+            <Scopes>internal_claim_meta_view</Scopes>
+        </Resource>
+        <Resource context="(.*)/api/server/v1/validation-rules(.*)" secured="true" http-method="PUT">
+            <Permissions>/permission/admin/manage/identity/applicationmgt/update</Permissions>
+            <Scopes>internal_application_mgt_update</Scopes>
```

Comment:
shouldn't these be claim mgt metadata related permissions ?

Reply:
Addressed with d2b87068136eaacb185ffdb3117c07cbed2924fd